common/config/rush/pnpm-config.json

```diff
@@ -1,192 +0,0 @@
-/**
- * This configuration file provides settings specific to the PNPM package manager.
- * More documentation is available on the Rush website: https://rushjs.io
- */
-{
-  "$schema": "https://developer.microsoft.com/json-schemas/rush/v5/pnpm-config.schema.json",
```

Comment:
Why are you deleting this file?

Reply:
Ah, got deleted during some build issue debugging. Revived.